lmn/models.py

```diff
@@ -64,7 +64,9 @@ def save(self, *args, **kwargs):
         if old_note and old_note.image:
             if old_note.image != self.image:
                 self.delete_image(old_note.image)
-
+        # note_count = Notes.objects.all().count()
```

Comment:
This should be cleaned up before merging. Everything's working for me though!

Reply:
You are correct. Changes made